### PR TITLE
magento/devdocs#: Cloud. Elastic search.

### PR DESCRIPTION
**Pull request by atwixfirster**
## Purpose of this pull request

This pull request (PR) adds a notice about ElasticSearch restart operation.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.2/cloud/project/project-conf-files_services-elastic.html
https://devdocs.magento.com/guides/v2.3/cloud/project/project-conf-files_services-elastic.html

## Links to Magento source code

Thank you!

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.2/cloud/project/project-conf-files_services-elastic.md [hunk 1]
```diff
@@ -55,6 +55,9 @@ functional_areas:
     bin/magento cache:clean
     ```
 
+{: .bs-callout-info }
+Not possible to restart the [Elasticsearch](https://www.elastic.co) service itself without Magento's help.
+
```
Comment (rogyar) [inline guides/v2.2/cloud/project/project-conf-files_services-elastic.md]: Please, adjust everywhere as “If you need to restart the elasticsearch service, you need to contact Magento support”

Comment (atwixfirster) [reply guides/v2.2/cloud/project/project-conf-files_services-elastic.md]: **done**

Comment (rogyar) [reply guides/v2.2/cloud/project/project-conf-files_services-elastic.md]: Thank you for the adjustments. The only place that left unchanged is this one ↑
Please, change this part (lines 58-60) as well. Thank you!

Comment (atwixfirster) [reply guides/v2.2/cloud/project/project-conf-files_services-elastic.md]: duplicated content has been removed . Thanks for notice 👍

File: guides/v2.2/cloud/project/project-conf-files_services-elastic.md [hunk 2]
```diff
@@ -92,5 +92,8 @@ See [Elasticsearch plugin documentation](https://www.elastic.co/guide/en/elastic
 {:.bs-callout-info}
 If you use the ElasticSuite third-party plugin, you must [update the `{{site.data.var.ct}}` package]({{page.baseurl}}/cloud/project/ece-tools-update.html) to version 2002.0.19 or later.
 
+{: .bs-callout-info }
+If you need to restart the [elasticsearch](https://www.elastic.co) service, you need to contact Magento support.
```
Comment (jeff-matthews) [inline guides/v2.2/cloud/project/project-conf-files_services-elastic.md]:
```suggestion
If you need to restart the [Elasticsearch](https://www.elastic.co) service, you must contact Magento support.
```

File: guides/v2.3/cloud/project/project-conf-files_services-elastic.md [hunk 3]
```diff
@@ -76,5 +76,8 @@ See [Elasticsearch plugin documentation](https://www.elastic.co/guide/en/elastic
 {:.bs-callout-info}
 If you use the ElasticSuite third-party plugin, you must [update the `{{site.data.var.ct}}` package]({{page.baseurl}}/cloud/project/ece-tools-update.html) to version 2002.0.19 or later.
 
+{: .bs-callout-info }
+If you need to restart the [elasticsearch](https://www.elastic.co) service, you need to contact Magento support.
```
Comment (jeff-matthews) [inline guides/v2.3/cloud/project/project-conf-files_services-elastic.md]:
```suggestion
If you need to restart the [Elasticsearch](https://www.elastic.co) service, you must contact Magento support.
```

File: guides/v2.2/cloud/project/project-conf-files_services-elastic.md (hunk 2 above)
Comment (atwixfirster) [reply guides/v2.2/cloud/project/project-conf-files_services-elastic.md]: **fixed**

File: guides/v2.3/cloud/project/project-conf-files_services-elastic.md (hunk 3 above)
Comment (atwixfirster) [reply guides/v2.3/cloud/project/project-conf-files_services-elastic.md]: **fixed**

Thank you, @jeff-matthews

**Pull request merged**